hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                                    Private
+                                {% endif %}
+                            </td>
+                             <td>
+                                {% if res|user_permission:request.user.pk|lower == "none" %}
+                                    <strong class="label-private">{{ res|user_permission:request.user.pk }}</strong>
+                                {% elif res|user_permission:request.user.pk|lower == "open access" %}
+                                    <strong class="label-public">{{ res|user_permission:request.user.pk }}</strong>
+                                {% else %}
+                                    {{ res|user_permission:request.user.pk }}
+                                {% endif %}
+                            </td>
+                            <td>
+                                <span onclick="remove_collection_item('{{ res.short_id }}')" data-form-id="form-favorite-{{ res.short_id }}" data-form-type="toggle-favorite"
+                                                  class="glyphicon glyphicon-remove btn-inline-favorite"></span>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                {% endif %}
+                </tbody>
+            </table>
+
+            {% if edit_mode %}
+                </form>
+                <input class="btn btn-primary" type="button" id="save-new" onclick="save_btn_callback_new();" value="Save Changes" style="display: none"/>
+            {% endif %}
+        </div>
+        {% if deleted_resources|length > 0 %}
+            <div id="view_deleted_res_div">
+                <input type="button" id="view_deleted_res_btn" style="color: red" value="View deleted contents" onclick=" $('#deleted-resources-modal').modal('show');">
+                <span data-toggle="tooltip" data-placement="auto"
+                      title='{% if edit_mode %}There may be broken resource links in the bag of this collection due to the deleted resources. You need to clear the deleted resources to fix that.{% else %}Some of the resources have been deleted from this collection.{% endif %}'
+                      class="glyphicon glyphicon-info-sign text-muted"></span>
+            </div>
+        {% endif %}
+        <hr>
+        {% block download_bag %}
+            <div>
+                {% for b in cm.bags.all %}
+                    <a id="btn-download-all" type="button" class="btn btn-default row-selector" href="{{ bag_url }}">
+                        <span class="glyphicon glyphicon-download-alt">
+                            <span title='Download All Content as Zipped BagIt Archive' class="button-label">Download Collection Bag</span>
+                        </span>
+                        <a target="_blank" href="http://en.wikipedia.org/wiki/BagIt">Learn more about the Bagit archive format</a>
+                    </a>
+                {% endfor %}
+            </div>
+        {% endblock %}
```

Comment:
@zhiyuli Got it, it is OK then, although it'd be better if we could remove repetitive code.